packages/@o3r/dev-tools/eslint.local.config.mjs

```diff
@@ -23,5 +23,11 @@ export default [
         ]
       }
     }
+  },
+  {
+    name: '@o3r/dev-tools/override',
+    rules: {
+      'unicorn/no-process-exit': 'off'
```

Comment:
This is the case for CLIs only (and maybe all the CLIs of the repo?)

Reply:
This package are only dev tools for CLI no ? and it is deprecated we will remove it in v12